From 3441f42d5c748bfa998c9cb28f9345a3ca7ba203 Mon Sep 17 00:00:00 2001
From: Minghuan Lian <minghuan.lian@nxp.com>
Date: Mon, 28 Oct 2024 19:01:46 +0800
Subject: [PATCH 7/7] yocto: not install header file

Real-time Edge has installed IgH package which includes
the same header files. IgH userspace package does not
need to install the herder files again. Otherwise, yocto
will report the error.

Signed-off-by: Minghuan Lian <minghuan.lian@nxp.com>
Upstream-Status: Pending
---
 include/Makefile.am | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/Makefile.am b/include/Makefile.am
index b5af07e..a879c4d 100644
--- a/include/Makefile.am
+++ b/include/Makefile.am
@@ -31,8 +31,6 @@
 #
 #------------------------------------------------------------------------------
 
-include_HEADERS = \
-	ecrt.h \
-	ectty.h
+include_HEADERS = 
 
 #------------------------------------------------------------------------------
-- 
2.25.1

